lib/GitHub/Apps/Auth.pm

```diff
@@ -157,10 +187,115 @@ sub issued_token {
     my $self = shift;
 
     if ($self->_is_expired_token) {
-        return $self->_prefix . $self->_fetch_access_token . $self->_suffix;
+        return $self->_fetch_access_token;
```

Comment:
`issued_token` joins a prefix and a suffix, and `_fetch_access_token` does it too.
the original test doesn't pass this line, so the test miss this bug.

```perl
# orignal test
my $joined_auth = "x-access-token:" . $auth;
$auth->token("1234567890");
is $joined_auth, "x-access-token:1234567890"; # it's ok.  $joined_auth is "x-access-token:1234567890"

# fixed test
sleep 61; # wait for expire token.
is $joined_auth, "x-access-token:1234567890"; # not ok.  $joined_auth is "x-access-token:x-access-token:1234567890"
```